index.js

```diff
@@ -32,34 +32,42 @@ function latex(src, options) {
    */
   const printErrors = (tempPath, userLogPath) => {
     const errorLogPath = path.join(tempPath, 'texput.log')
-    const errorLogStream = fs.createReadStream(errorLogPath)
 
-    if (userLogPath) {
-      const userLogStream = fs.createWriteStream(path.resolve(userLogPath))
-      errorLogStream.pipe(userLogStream)
-    }
+    fs.stat(errorLogPath, (err, stats) => {
+      if (err || !stats.isFile()) {
+        outputStream.emit('error', new Error('LaTeX Syntax Error'))
```

Comment:
nit: Can you change error message to be something more along the lines of the error log file not existing?

Reply:
Like using `err` variable?

Reply:
sorry, i meant instead of `LateX Syntax Error` it should be a different message like `No error log file` or something

Reply:
Here you go! `No error log file` was perfect.